packages/devtools-components/src/tree.js

```diff
@@ -12,6 +12,20 @@ require("./tree.css");
 // depth
 const AUTO_EXPAND_DEPTH = 0;
 
+// Simplied selector targetting elements that can receive the focus, full
+// version at
+// http://stackoverflow.com/questions/1599660/which-html-elements-can-receive-focus
+// .
```

Comment:
this bit is a bit weird :)

Reply:
Ah that's taken from:
https://github.com/mozilla/gecko-dev/blob/master/devtools/client/shared/focus.js

Reply:
weird!